plugins/modules/rabbitmq_user.py

```diff
@@ -106,6 +106,33 @@
     required: false
     default: on_create
     choices: ['on_create', 'always']
+  protocol:
+    description:
+      - Specify which TCP/IP protocol will be used
+    type: str
+    default: http
+    choices: ['http', 'https']
+  host:
+    description:
+      - Hostname from API
+    type: str
+    required: false
+  port:
+    description:
+      - Port of access from API
+    type: str
+    required: false
+  login_user:
+    description:
+      - User from administrator of the management API
```

Comment:
```suggestion
      - User from administrator of the management API.
```
Also maybe `User from administrator` -> `Administrator's username` or `Username with administrator access`?